README.md

```diff
@@ -13,9 +13,8 @@
 ## Installation
 
 ```bash
-git clone https://github.com/ArkEcosystem/python-client
-cd python-client
-sudo python3 setup.py install
+pip3 install setuptools
+pip3 install https://github.com/ArkEcosystem/python-client/archive/develop.zip
```

Comment:
instead of that you can just do `pip3 install git+git://github.com/ArkEcosystem/python-client.git`